app/models/order.rb

```diff
@@ -12,8 +12,13 @@ def updated_at_formatted
   end
 
   def cart_item_and_quantity
-    items = Hash.new
-    cart_items.select { |item_id, quantity| items[Item.find(item_id)] = quantity }
-    items
+    @items = Hash.new
+    cart_items.select { |item_id, quantity| @items[Item.find(item_id)] = quantity }
+    @items
+  end
+
+  def total_price
+    prices = @items.map { |item, quantity| item.price }
```

Comment:
Unused block argument - `quantity`. If it's necessary, use `_` or `_quantity` as an argument name to indicate that it won't be used.